Create CollaborationsWorkshop2020-submission.md (#816)
alan-turing-institute/the-turing-way: conferences/abstracts/CollaborationsWorkshop2020-submission.md

# Collaborations Workshop 2020

* Website: https://www.software.ac.uk/cw20
* Application form: https://www.eventbrite.co.uk/register
* Location: Belfast, Northern Ireland
* Date: 31 March - 2 April 2020

### Title

* The Turing Way: a "lightly-opinionated" guide to data science and research

### Authors

* Malvika Sharan with Sarah Gibson and Jez Cope on behalf of **The Turing Way Community**

### Abstract

The *Turing Way* is an open source community-driven and collaboratively-developed book project on data science and research. 

> Launched and demo-ed at the (Software Sustainability Institute's Collaborations Workshop](https://www.software.ac.uk/cw19) last year, the *Turing Way* aimed at supporting researchers, educators, funders and journal editors in ensuring that reproducible data science is "too easy not to do".

Reviewer: ```suggestion
Launched and demo-ed at the [Software Sustainability Institute's Collaborations Workshop](https://www.software.ac.uk/cw19) last year, the *Turing Way* aims to support researchers, educators, funders and journal editors in ensuring that reproducible data science is "too easy not to do".
```

> Since its launch, the *Turing Way* has gained interests from a diverse set of community members working across different disciplines within data science research and training, who have contributed to [the project repository](https://github.com/alan-turing-institute/the-turing-way) through in-person contribution workshops (book dash events) and online co-working calls (collaboration cafes). Furthermore, the *Turing Way* team members have delivered workshops on GitHub, git, Jupyter Notebook, BinderHub, Binder, and basic programming to impart skills and confidence to empower new contributors, and thus, to sustain an open-source ecosystem.

Reviewer: ```suggestion
Since its launch, *The Turing Way* has gained interest from a diverse set of community members working across different disciplines within data science research and training, who have contributed to [the project repository](https://github.com/alan-turing-institute/the-turing-way) through in-person contribution workshops (book dash events) and online co-working calls (collaboration cafes). 
Furthermore, the *Turing Way* team members have delivered workshops on GitHub, git, Jupyter Notebook, BinderHub, Binder, and basic programming to impart skills and confidence to empower new contributors, and thus, to sustain an open-source ecosystem.
```

> Going forward, *The Turing Way* project aims to expand into a series of books, which, in addition to reproducibility, will capture the vast amounts of knowledge of data science practices in the areas of project design, communication and outreach, collaboration, and ethics.  

Reviewer: ```suggestion
Going forward, *The Turing Way* project will expand into a series of books, which, in addition to reproducibility, will capture the vast amounts of knowledge of data science practices in the areas of project design, communication and outreach, collaboration, and ethics.  
```

> In this proposed session, we will introduce the project in a short talk on *Turing Way* followed by an interactive demonstrations of the existing and planned chapters that participants can use, share, develop or modify for their own research. All participants will leave the session knowing that "Every Little Helps" when making their work intentionally designed to be open and reproducible. The session facilitators will help participants in identifying their route of contribution to an open source project while improving the *Turing Way* handbook for future readers.

Reviewer: ```suggestion
In this proposed session, we will introduce the project in a short talk on *The Turing Way* followed by an interactive demonstrations of the existing and planned chapters that participants can use, share, develop or modify for their own research.
All participants will leave the session knowing that "Every Little Helps" when making their work intentionally designed to be open and reproducible.
The session facilitators will help participants in identifying their personal route of contribution to an open source project while improving *The Turing Way* handbook for future readers.
```
